core/src/main/scala/cats/syntax/parallel.scala

```diff
@@ -32,8 +32,12 @@ trait ParallelSyntax extends TupleParallelSyntax {
   implicit final def catsSyntaxParallelSequence1[T[_]: Traverse, M[_], A](tma: T[M[A]]): ParallelSequenceOps1[T, M, A] =
     new ParallelSequenceOps1(tma)
 
+  @deprecated("Kept for binary compatibility", "2.8.0")
   implicit final def catsSyntaxParallelAp[M[_]: FlatMap, A](ma: M[A]): ParallelApOps[M, A] =
```

Comment:
```suggestion
  @deprecated("Kept for binary compatibility", "2.8.0")
  private[syntax] final def catsSyntaxParallelAp[M[_]: FlatMap, A](ma: M[A]): ParallelApOps[M, A] =
```

Reply:
I think you reverted this because MiMa was complaining about missing static methods? I think it is is safe to filter those, because they only affect Java and I think it would be unusual for Java code to be using these syntax extensions.

Reply:
I think it may be okay to leave it public if `private[syntax]` breaks BC. The only thing – the `implicit` modifier should be pulled off the identifier, but as I can see, it is already done here.